vision_datasets/common/data_manifest.py

```diff
@@ -742,6 +766,12 @@ def create_dataset_manifest(coco_file_path_or_url, data_type, container_sas_or_r
             images = [x for x in images_by_id.values()]
             return DatasetManifest(images, None, data_type)
 
+        if data_type == DatasetTypes.IMAGE_MATTING:
+            for annotation in coco_manifest['annotations']:
+                images_by_id[annotation['image_id']].label_file_paths = [get_full_sas_or_path(annotation['label'])]
```

Comment:
if 'image_id' exists, should append instead of overwritting existing array

Reply:
Resolved!